Core/include/Acts/Definitions/PdgParticle.hpp

```diff
@@ -41,4 +45,80 @@ static constexpr PdgParticle makeAbsolutePdgParticle(PdgParticle pdg) {
   return static_cast<PdgParticle>((0 <= value) ? value : -value);
 }
 
+/// Check if the PDG belongs to a nucleus, i.e. if it has 10 digits.
+/// See PDG section "Monte Carlo Particle Numbering Scheme", point 16:
+/// https://pdg.lbl.gov/2025/reviews/rpp2024-rev-monte-carlo-numbering.pdf
+static constexpr bool isNucleus(PdgParticle pdg) {
+  const auto pdgNum = static_cast<std::int32_t>(pdg);
+  return std::abs(pdgNum) > 1e9;
+}
+
+/// Convert an excited nucleus to its ground state. PDG number of a nucleus has
+/// a form 10LZZZAAAI, where I is isomer level; I=0 is the ground state. Leave
+/// other particles as-is. See PDG section "Monte Carlo Particle Numbering
+/// Scheme", point 16:
+/// https://pdg.lbl.gov/2025/reviews/rpp2024-rev-monte-carlo-numbering.pdf
+static constexpr PdgParticle makeNucleusGroundState(PdgParticle pdg) {
+  if (isNucleus(pdg)) {
+    const auto pdgNum = static_cast<std::int32_t>(pdg);
+    // set isomer level to zero
+    return static_cast<PdgParticle>((pdgNum / 10) * 10);
+  } else {
+    return pdg;
+  }
```

Comment:
I would argue that we can throw an exception if the PDG does not represent a nucleus. I believe this is possible even if the function is `constexpr`?
```suggestion
  if (!isNucleus(pdg)) {
    throw std:: invalid_argument("pdg must be a nucleus");
  }

  const auto pdgNum = static_cast<std::int32_t>(pdg);
  // set isomer level to zero
  return static_cast<PdgParticle>((pdgNum / 10) * 10);
```

Reply:
I agree; yes, it's possible to throw exception in `constexpr`